scriptured-prayer-components/src/components/PrayerDeck.tsx

```diff
@@ -45,11 +50,18 @@ function PrayerDeck() {
           pagination={{ clickable: true }}
           keyboard
         >
-          {cards.map((card) => (
-            <SwiperSlide key={card.id}>
-              <Card {...card} />
-            </SwiperSlide>
-          ))}
+          {category && (
+            <>
+              <SwiperSlide>
+                <Card {...category} />
+              </SwiperSlide>
+              {cards.map((card) => (
+                <SwiperSlide key={card.id}>
+                  <Card {...card} />
```

Comment:
Supporting my other comment regarding an updated `props` type:

```suggestion
                <Card
                  focus
                  category={category.name}
                  title="Inspiration"
                  body={category.inspiration}
                  cardCount={category.card_count}
                />
              </SwiperSlide>
              {cards.map((card) => (
                <SwiperSlide key={card.id}>
                  <Card
                    category={card.category}
                    title={card.title}
                    body={card.scripture_text.map((st) => st.text).join(" ")}
                  />
```